extensions/wikia/SpecialDiscussions/templates/SpecialDiscussions_index.mustache

```diff
@@ -0,0 +1,8 @@
+<div class="discussions-container">
+    {{#discussionsLink}}
```

Comment:
This is mustache's version of `if`/`else` and in the index method above you also have an `if`/`else` in PHP for the same condition.  I would suggest just keeping one of them and I would probably keep the one in PHP.  You could replace all of this with a simple `{{{content}}}` include